cpp/map_module/algorithm/map.hpp

```diff
@@ -1,5 +1,18 @@
 #pragma once
 
 #include <mgp.hpp>
+#include <string_view>
 
-namespace Map {}  // namespace Map
+namespace Map {
+
+constexpr const std::string_view kProcedureSetKey = "setKey";
```

Comment:
I think it makes sesne to call this function `set_key` as this will be consistent with rest of our functions